tests/eips/eip4844/test_excess_data_gas.py

```diff
@@ -507,8 +513,8 @@ def test_invalid_static_excess_data_gas(
 
     Test is parametrized to `MAX_BLOBS_PER_BLOCK` and `TARGET_BLOBS_PER_BLOCK`.
     """
-    blocks[0].rlp_modifier = Header(excess_data_gas=parent_excess_data_gas)
-    blocks[0].exception = "invalid excessDataGas"
+    blocks[1].rlp_modifier = Header(excess_data_gas=parent_excess_data_gas)
+    blocks[1].exception = "invalid excessDataGas"
```

Comment:
```suggestion
    blocks[-1].rlp_modifier = Header(excess_data_gas=parent_excess_data_gas)
    blocks[-1].exception = "invalid excessDataGas"
```
To always use the last block.